dune_client/api/extensions.py

```diff
@@ -316,40 +317,6 @@ def download_csv(
             ),
         )
 
-    ############################
```

Comment:
I think some people might still be depending on this. We might need to keep it for backwards compatibility

Reply:
I was moved here 
https://github.com/duneanalytics/dune-client/pull/114/files/dd6d30d78502ea752bd6c04cc1ea7299da2e1347#diff-480e443b4b14d9d3ff5327a749050ff3e17aa60adbdd304fa90b524d89644073R19